src/cider/nrepl/middleware/info.clj

```diff
@@ -22,41 +22,41 @@
   [ns sym]
   (or
     ;; it's a special (special-symbol?)
-    (m/special-sym-meta sym)
+   (m/special-sym-meta sym)
```

Comment:
Another case of misaligned code and comments.

Reply:
again, weavejester/cljfmt#112.